sphinx/builders/__init__.py

```diff
@@ -247,12 +248,14 @@ def cat2relpath(cat):
 
     def compile_all_catalogs(self):
         # type: () -> None
+
         catalogs = i18n.find_catalog_source_files(
             [path.join(self.srcdir, x) for x in self.config.locale_dirs],
             self.config.language,
             charset=self.config.source_encoding,
             gettext_compact=self.config.gettext_compact,
-            force_all=True)
+            force_all=True,
+            excluded=Matcher(['**/.?**']))
```

Comment:
We have three calls of `find_catalog_source_files()` in this file.
Please add `excluded` argument to other calls.

Reply:
done.